packages/fluentui/react-northstar/src/themes/teams-dark/components/SplitButton/splitButtonVariables.ts

```diff
@@ -0,0 +1,8 @@
+import { SplitButtonVariables } from '../../../teams/components/SplitButton/splitButtonVariables';
+
+export const splitButtonVariables = (siteVars: any): Partial<SplitButtonVariables> => {
+  return {
+    toggleButtonBorderColor: 'red', // test
```

Comment:
Can anyone help me understand why these values aren't being picked up when I switch themes?

Reply:
@codepretty you need to also create a `splitButtonToggleVariables.ts` file and export it from `componentVariables` in each theme... here are the code links to help you out:

* [splitButtonToggleVariables.ts](https://github.com/microsoft/fluentui/blob/master/packages/fluentui/react-northstar/src/themes/teams/components/SplitButton/splitButtonToggleVariables.ts)
* [componentVariables.ts](https://github.com/microsoft/fluentui/blob/bd901a7dc2ee73d7d85d841e0c547320322035e1/packages/fluentui/react-northstar/src/themes/teams/componentVariables.ts#L146)

Reply:
Makes total sense, thanks for the help!